src/t8_eclass.h

```diff
@@ -153,6 +153,11 @@ int                 t8_eclass_compare (t8_eclass_t eclass1,
 */
 int                 t8_eclass_is_valid (t8_eclass_t eclass);
 
+/** Returns true, if an element does not refine into 2^dim children.
+ * Returns false otherwise.
+*/
+int                 t8_eclass_refines_irregular (t8_eclass_t eclass);
+
```

Comment:
see comment, this function does not belong to the eclass interface.

Reply:
Done